tests/aws/services/dynamodb/test_dynamodb.py

```diff
@@ -800,7 +800,9 @@ def test_dynamodb_stream_stream_view_type(self, aws_client):
         dynamodb.delete_table(TableName=table_name)
 
     @markers.aws.only_localstack
-    def test_dynamodb_with_kinesis_stream(self, aws_client, secondary_aws_client):
+    def test_dynamodb_with_kinesis_stream(
+        self, aws_client, secondary_aws_client, wait_for_stream_ready
```

Comment:
minor: `wait_for_stream_ready` is not used in this test. It should probably replace new lines 813-814

Reply:
@pinzon can you give some context here? Do we know why this is marked `only_localstack`?

Reply:
I guess it's related with the secondary account necessary to test the cross-account integration.